android/src/main/java/co/nimblehq/ic/kmm/suv/android/ui/screens/home/views/HomeSurveysView.kt

```diff
@@ -111,28 +126,54 @@ private fun HomeSurveysContent(uiModel: HomeSurveysUiModel) {
                     .fillMaxSize()
                     .padding(horizontal = AppTheme.dimensions.mediumPadding)
             ) {
-                DotsIndicator(uiModel.totalPages, uiModel.currentPageIndex)
-                Spacer(modifier = Modifier.height(AppTheme.dimensions.mediumPadding))
-                Text(
-                    text = uiModel.currentSurveyUiModel.title,
-                    color = Color.White,
-                    style = Typography.h5,
-                    maxLines = 4
+                DotsIndicator(
+                    uiModel.totalPages,
+                    uiModel.currentPageIndex,
+                    modifier = Modifier
+                        .semantics { contentDescription = HomeContentDescription.INDICATOR }
                 )
+                Spacer(modifier = Modifier.height(AppTheme.dimensions.mediumPadding))
+                AnimatedContent(
```

Comment:
Following the animation on Figma, I think we should use `CrossFade` instead, I am not sure if you tried it yet 😄 
https://developer.android.com/jetpack/compose/animation#crossfade

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/107/commits/beae7674d690e5854ce56b7a98b4577a2dec5885